README.md

```diff
@@ -6,7 +6,86 @@ This repo contains the pairing test exercises that are used in the Guardian recr
+
+The candidate should be the driver in the pairing exercise (especially so if you are not  familiar with the language).
+
+In a few cases, it may be applicable for you to drive and the candidate to navigate. For example if the candidate has no knowledge about the language they have chosen. This should be rare.
+
+## What exercise?
+[From: The Guardian's new pairing exercises](https://www.theguardian.com/info/developer-blog/2016/jan/20/the-guardians-new-pairing-exercises)
+> ...we have decided to increase the number of pairing exercises, any of which can be picked by a Guardian developer prior to the pairing test.
+
+We have a public repository on GitHub with our pairing exercises. These exercises are used for every role.
+
+We have [this repository](https://github.com/guardian/pairing-test-project) on GitHub with skeleton projects that can be used.
+
+## Before the interview
+Choose a pairing exercise, checkout the [skeleton project repository](https://github.com/guardian/pairing-test-project), prepare your machine and setup up your IDE.
+
+Turn off notifications and other possible distractions for the candidate; an easy way to do this is to use a different browser profile and closing your work profile.
+
+Get some pen and paper in case the candidate (or you) need to draw anything.
+
+The developer manager for the role will collect the candidate from reception and bring them to your desk.
+
+Offer the candidate a drink.
+
+Explain what you’re going to be doing today.
+> This is the pairing test stage of the interview process. We’ll spend about 45 - 60 minutes writing code to solve a problem. We’re not assessing you on your deep technical knowledge or your understanding of the standard library, we’re more interested in how you solve a problem. With that in mind, feel free to search online for anything, ask questions etc. It’s not about how far we get through the exercise.
+
+## During the interview
+Provide guidance on where to start. For example, directory structure, where to add tests etc.
+
+Try not to touch the keyboard or dictate a solution as this doesn’t provide much detail about the candidate.
+
+If the candidate is struggling, guide the candidate to a solution. This can take many forms, such as diagramming, pseudo code etc.
+
+If you see a really obvious mistake, don’t let the candidate struggle with it. For example, if they’ve misspelt a variable, passed arguments to a function in the wrong order etc.
+
+Ask why the candidate is doing things that way to help understand their thought process.
+
+## After the interview
+Write up some notes about the interview focusing on the [assessment criteria](https://docs.google.com/spreadsheets/d/1Pyr6pDMvisGAYPCOAAS1YDIrA_GXvCE4YUc3c-vk0H4/edit#gid=0). You may also find it useful to add some inline comments to the code. These will be useful in the wash-up for candidate, where the interviewers discuss the candidate as a whole and decide if an offer should be made or not.
```

Comment:
It might be worth linking to https://developers.theguardian.com/open-people.html so they can see the criteria from the career progression framework

Reply:
```suggestion
Write up some notes about the interview focusing on the [assessment criteria](https://docs.google.com/spreadsheets/d/1Pyr6pDMvisGAYPCOAAS1YDIrA_GXvCE4YUc3c-vk0H4/edit#gid=0).
Candidates can see what is expected at each level in the [people section](https://developers.theguardian.com/open-people.html) of our developers site.
Interviewers may also find it useful to add some inline comments to the code. These will be useful in the wash-up for candidate, where the interviewers discuss the candidate as a whole and decide if an offer should be made or not.
```